pom.xml

```diff
@@ -12,6 +12,7 @@
     <properties>
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
+        <lombok.mapstruct.binding.version>0.2.0</lombok.mapstruct.binding.version>
```

Comment:
why we need it?

Reply:
I thought this was telling Maven which version of Java to use. If it is not necessary, I will delete it) Thank you!